tests/test___init__.py

```diff
@@ -143,8 +143,8 @@ def test_Majaj2015ITpls(self, model_identifier, expected_score, attach_hook):
         assert score.raw.sel(aggregation='center') == approx(expected_score, abs=0.005)
 
     @pytest.mark.parametrize(['model_identifier', 'expected_score'], [
-        ('CORnet-S', .25),
-        ('CORnet-R2', .224),
+        ('CORnet-S', .240888),
```

Comment:
I don't really understand why this score changed. With model's field of view of 8 degrees being identical to the stimulus presentation at 8 degrees, I thought it would remain unchanged.